test/test_configuration.rb

```diff
@@ -1,23 +1,25 @@
-require 'helper'
+# rubocop:disable Metrics/ClassLength
```

Comment:
I believe we can disable just this metric from `.rubocop.yml`

If you don’t mind, I’ll give it a shot